kodata/samples/buildstrategy/sample-strategies.yaml

```diff
@@ -684,94 +686,6 @@ spec:
     runAsUser: 1001
     runAsGroup: 1000
 
----
-# This Build Strategy will intentionally fail if the image has any
-# critical CVEs. It will not be pushed into the destination registry
-# if any critical vulnerabilities are found.
-apiVersion: shipwright.io/v1beta1
-kind: ClusterBuildStrategy
-metadata:
-  name: kaniko-trivy
```

Comment:
Noted in the release note that current users should switch to `kaniko` with the new vulnerability scan feature enabled.